client/src/components/components.ts

```diff
@@ -42,12 +42,12 @@ async function zeroDesignSystemImport(): Promise<DesignSystemModule> {
     );
     type = ResourceType.local;
     module = await import(
-      /* webpackChunkName: "foundation-zero" */
-      '@genesislcap/foundation-zero'
+      /* webpackChunkName: "rapid-design-system" */
+      '@genesislcap/rapid-design-system'
     );
     return assureDesignSystem(module);
   } finally {
-    logger.debug(`Using '${type}' version of foundation-zero`);
+    logger.debug(`Using '${type}' version of rapid-design-system`);
```

Comment:
Not related to this PR: this boilerplate for attempting to load a federated DS - we keep adding it to all projects even though no one's loading their DSes via federation (as far as I know). Not sure whether it needs to be enabled by default or could be a page in the docs explaining how to enable it if you want to.

Reply:
Removed...